### ISSUE TITLE
Fatal error: Call to undefined function ssh2_connect() in ...

### ISSUE THREAD

**Issue by Monter** [Rudloff/WebSSH#1]
Is this requires SSH2 PECL extension?

**Comment by Rudloff** [comment]
Yes, it requires the ssh2 extension (php5-ssh2 on Debian-based systems).
I added a composer.json that indicates the requirement (https://github.com/Rudloff/WebSSH/commit/d1fa4a9850d788f52b3b15483043d607d062f545).

We could detect if the extension is installed and display a warning if it is not, but I don't really have the time right now.
